integration/ldap/general_ldap_example.yaml

```diff
@@ -0,0 +1,5 @@
+---
+Group 1: # Name of group in GLPI
+  ldap: group1-users # Name of group in LDAP
+Group 1 > Group 2:
```

Comment:
Why use the angle bracket here, just for a newline? Does this mean both Group 1 and Group 2 of GLPI are mapped to group2-users in LDAP?

Reply:
This is how descendant groups are denoted in GLPI. So in this example group 2 is a subset of group 1, and group2-users will be mapped to only Group 2 of GLPI. I've added a clarifying comment to the YAML

Reply:
Got it, makes sense. Yeah I think the clarification is good